lib/grpc/client/adapters/mint.ex

```diff
@@ -114,12 +114,16 @@ defmodule GRPC.Client.Adapters.Mint do
       |> Keyword.get(:transport_opts, [])
       |> Keyword.merge(ssl)
 
-    [transport_opts: Keyword.merge(@default_transport_opts, transport_opts)]
+    keep_alive = Keyword.get(opts, :keep_alive, false)
+
+    [transport_opts: Keyword.merge(@default_transport_opts, transport_opts), keep_alive: keep_alive]
```

Comment:
For clarity, it's better if you add a question mark to this option, since, usually `keep_alive` is a provided as a number by convention. 


```suggestion
    keep_alive? = Keyword.get(opts, :keep_alive?, false)

    [transport_opts: Keyword.merge(@default_transport_opts, transport_opts), keep_alive?: keep_alive?]
```

Reply:
I wonder if it's valuable to add a module doc describing this adapter-specific option. WDYT?

Reply:
https://github.com/elixir-mint/mint/blob/main/lib/mint/http.ex#L276-L364
can't see any keep_alive option here

Reply:
`transport_opts` is forwarded to `gen_tcp` or `ssl`. The Mint docs list the most common ones, but there are more of them.

https://www.erlang.org/doc/man/gen_tcp.html#type-option

Now that you mentioned, `keepalive` on gen_tcp doesn't have a underscore